src/utils/__tests__/gameStateManager.test.ts

```
@@ -0,0 +1,301 @@
+const mockTestResult = {
+  id: 'test-id',
+  imageUri: 'test.jpg',
+  trueLabel: 'apple' as const,
+  predictedLabel: 'apple' as const,
+  confidence: 0.9,
+  isCorrect: true,
+  predictionTime: 500,
+};
+
+import { GameState } from '@/types';
+import { GameState } from '@/types';
```

Biome (2.1.2)

[error] Shouldn't redeclare 'GameState'. Consider to delete it or rename it.

'GameState' is defined here:


(lint/suspicious/noRedeclare)